docs/features.md

```diff
@@ -0,0 +1,315 @@
+    "resources": [
+      {
+        # matches a resource with label "foo" equal to "bar"
+        "match": {
+          "$.metadata.labels.foo": "bar"
+        },
+        # message to display in the tooltip
+        "message": "This resource is protected"
+      },
+      {
+        # matches a resource with both label "foo" equal to "bar" and label "baz" equal to "qux"
+        "match": {
+          "$.metadata.labels.foo": "bar",
+          "$.metadata.labels.baz": "qux"
+        },
+        # source of the message to display in the tooltip
+        "messageSrc": "$.metadata.annotations.protected-message"
+      },
+      {
+        "match": {
+          "$.metadata.labels.foo": "bar"
+        }
+        # if neither "message" or "messageSrc" is given, a generic message will be used (from i18n files - key: common.protected-resource)
+      }
+    ]
+  }
+}
+```
+
+The match keys and messageSrc must use format as described at
+<https://github.com/dchester/jsonpath>.
+
+- **SENTRY** – is used to enable monitoring of uncaught exceptions, which then are analyzed and repaired. The address to which you send the information is located under the dsn key.
+
+Default settings:
+
+```bash
+"SENTRY ":{
+  "isEnabled": false,
+  "selectors":[ ],
+  "config": {
+    "dsn":""
+  }
+},
+```
+
+- **SERVERLESS** – is used to show or hide the Serverless view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.  
+  For the view to be shown, you must enable the feature and all the CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"SERVERLESS ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"serverless.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **SERVICE_CATALOG** – is used to show or hide the Service Catalog views (Catalog, Instances, and Brokers) as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.  
+  SERVICE_CATALOG and SERVICE_CATALOG_ADDONS are also used to determine if Service Bindings (in Configuration tab) and environment variables injected by Service Bindings (in Code Tab) are displayed in the Functions view.  
+  For the view to be shown, you must enable the feature and all the CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"SERVICE_CATALOG":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"servicecatalog.k8s.io"
+    }
+  ]
+},
+```
+
+- **SERVICE_CATALOG_ADDONS** – is used to show or hide the Service Catalog Addons view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.  
+  SERVICE_CATALOG and SERVICE_CATALOG_ADDONS are also used to determine if Service Bindings (in Configuration tab) and environment variables injected by Service Bindings (in Code Tab) are displayed in the Functions view.  
```

Comment:
The info is duplicated. See the suggestion above. Maybe you could remove the one above and make a NOTE out if it here?